lib/spack/spack/bootstrap.py

```diff
@@ -758,10 +783,23 @@ def gnupg_root_spec():
 
 def ensure_gpg_in_path_or_raise():
     """Ensure gpg or gpg2 are in the PATH or raise."""
-    ensure_executables_in_path_or_raise(
+    return ensure_executables_in_path_or_raise(
         executables=['gpg2', 'gpg'], abstract_spec=gnupg_root_spec()
     )
 
+
+def patchelf_root_spec():
+    """Return the root spec used to bootstrap GnuPG"""
+    return _root_spec('patchelf')
+
+
+def ensure_patchelf_in_path_or_raise():
+    """Ensure gpg or gpg2 are in the PATH or raise."""
```

Comment:
```suggestion
    """Ensure patchelf is in the PATH or raise."""
```
Or just delete the docstring cause it's not adding any new information given the function name.

Reply:
Would you prefer to fix this or delete all of them?

Reply:
For me it's the same, if it's less work to hit commit suggestion feel free ;p